src/utils/renderSchema.ts

```diff
@@ -104,10 +104,13 @@ export const renderSchema: Walker = function*(schema, level = 0, meta = { path:
       if (node.properties !== undefined) {
         for (const [i, property] of node.properties.entries()) {
           if ('type' in node) {
-            property.type = property.type || node.type;
+            node.properties[i] = {
```

Comment:
So is this needed in general? Feels like it might not be.

Reply:
Yeah, it's needed. If you revert it, a few tests covering mutation will fail.

Reply:
Are you sure? For me only two tests fail, both snapshots regarding the "type": "object" thing. Not saying that means everything is good, but just that the tests don't seem to indicate any big issues?

<img width="798" alt="Screen Shot 2019-10-15 at 6 58 00 PM" src="https://user-images.githubusercontent.com/847542/66878012-961dda00-ef7d-11e9-8d22-f98fb3fbea0a.png">

Reply:
>  For me only two tests fail,

Yup, these are the only ones, since they cover the combiner children case.

> Not saying that means everything is good, but just that the tests don't seem to indicate any big issues?

That's right, seems like this was the only spot.